tox.ini

```
@@ -36,7 +36,7 @@ deps =
     testing.postgresql
     testing.mysqld
     webtest
-    mock
+    mock ; python_version < '3.3'
 
     # Python2 only deps
     py{27}: enum34
```

Maybe we can move the dependency to the `Python2 only deps` section to be consistent?

```suggestion
    # Python2 only deps
    py{27}: enum34, mock
```